src/main/java/org/jabref/gui/search/GlobalSearchBar.java

```diff
@@ -126,13 +126,14 @@ public GlobalSearchBar(JabRefFrame frame, StateManager stateManager, Preferences
         // searchModeButton = new Button();
         initSearchModifierButtons();
 
-        BooleanBinding focusBinding = searchField.focusedProperty()
-                                                 .or(regularExpressionButton.focusedProperty()
-                                                                            .or(caseSensitiveButton.focusedProperty()));
```

Comment:
One nitpick, you need to re-add the `focusedProperty()` of `regularExpressionButton` and `caseSensitiveButton` with the new changes. Otherwise, they will disappear if you click on them while the search field is empty.

Sorry about making your life difficult =/ 

Except that it looks really good. I like this behaviour 😃

Reply:
No worries!
Thank you for your feedback ^^
I now added the focused properties back in 😃!